### DISCUSSION TITLE
Import/Sync with PipePipe

### DISCUSSION THREAD

**Discussion by chris02000** [FreeTubeApp/FreeTube#5655] [Q&A]
Hi,
Love freetube thanks.
Is there a way to import everything from pipepipe or even sync the sync the two?

Many tthanks
Chris

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi Chris, we do not support sync with PipePipe. Also PipePipe is a hard fork of NewPipe so it could be that the subscription import is working but i cant guarantee for certain that it will work.

**Reply by chris02000** [reply]
Thanks, keep the great work.
